README.md

```diff
@@ -1,9 +1,13 @@
 ## EKS Distro Repository
 
+### 🚨🚨🚨 IMPORTANT INFORMATION ABOUT 1.22 SUPPORT 🚨🚨🚨
```

Comment:
Do we need this section here? It’s been here for several weeks.

Reply:
@kschumy I thought we are announcing that we have stopped supporting 1.22, but if we are thinking SNS as the announcement, I can get rid of it.